### PR TITLE
build rust runtime for dragonball hypervisor

### PR DESCRIPTION
**Pull request by yqleng1987**
Signed-off-by: yqleng <yqleng@linux.alibaba.com>

### CODE REVIEW

File: .ci/install_kata.sh [hunk 1]
```diff
@@ -63,6 +63,9 @@ case "${KATA_HYPERVISOR}" in
 		echo "Installing virtiofsd"
 		"${cidir}/install_virtiofsd.sh"
 		;;
+	"dragonball")
+		echo "Kata Hypervirsor is dragonball"
```
Comment (GabyCT) [inline .ci/install_kata.sh:67]: s/Hypervirsor/Hypervisor

**Comment by GabyCT** [comment]
Is there a CI that will run this?

**Review by GabyCT — APPROVED** [review]

File: .ci/install_kata.sh (hunk 1 above)
Comment (yqleng1987) [reply .ci/install_kata.sh]: Thanks. Already fix the typo.

**Comment by yqleng1987** [comment]
> 

Yes, CI will run for rund. Currently we are working on the adaption between the "runtime-rs" branch in "tests" repo and the "runtime-rs" branch in "kata-containers" repo

**Comment by yqleng1987** [comment]
> Is there a CI that will run this?

Yes, CI will run for rund. Currently we are working on the adaption between the "runtime-rs" branch in "tests" repo and the "runtime-rs" branch in "kata-containers" repo

**Review by liubin — APPROVED** [review]

**Pull request merged**